config/routes.rb

```diff
@@ -355,34 +355,55 @@
                dialog_runner_post
     },
 
-    :chargeback               => {
+    :chargeback_assignment => {
       :get  => %w(
         explorer
-        render_csv
-        render_pdf
-        render_txt
-        report_only
       ),
       :post => %w(
-        accordion_select
         explorer
         cb_assign_field_changed
         cb_assign_update
+        tree_select
+      )
+    },
+    :chargeback_rate => {
+      :get  => %w(
+        explorer
+        x_show
+      ),
+      :post => %w(
+        explorer
         cb_rate_edit
         cb_rate_form_field_changed
         cb_rate_show
         cb_rates_delete
         cb_rates_list
         cb_tier_add
         cb_tier_remove
-        saved_report_paging
         tree_autoload
         tree_select
         x_button
         x_show
       )
     },
 
+    :chargeback_report => {
+      :get  => %w(
+        explorer
+        render_csv
+        render_pdf
+        render_txt
+        report_only
+      ),
+      :post => %w(
+        explorer
+        saved_report_paging
+        tree_autoload
```

Comment:
This is not necessary, in any of them.

Reply:
There's no lazyloading in the trees.

Reply:
@skateman chargeback_reports tree does autoloading of tree, removing the route for those throws an error
```
[----] I, [2020-05-12T10:46:27.924024 #16852:2f036a4]  INFO -- : Started POST "/chargeback_report/tree_autoload" for 127.0.0.1 at 2020-05-12 10:46:27 -0400
[----] F, [2020-05-12T10:46:27.961290 #16852:2f036a4] FATAL -- :   
[----] F, [2020-05-12T10:46:27.961391 #16852:2f036a4] FATAL -- : ActionController::RoutingError (No route matches [POST] "/chargeback_report/tree_autoload"):
[----] F, [2020-05-12T10:46:27.961439 #16852:2f036a4] FATAL -- :   
[----] F, [2020-05-12T10:46:27.961496 #16852:2f036a4] FATAL -- : actionpack (5.2.4.2) lib/action_dispatch/middleware/debug_exceptions.rb:65:in `call'
```

Reply:
Weird, I'll address that in a future PR.